modules/system/effect-wfrp4e.js

```diff
@@ -691,8 +717,7 @@ export default class EffectWfrp4e extends ActiveEffect
             areaType : "sustained", // Area - "instantaneous" or "sustained"
             renderAura : true, // Whether or not to render the measured template
 
-            targetedAura : false, // Aura - if the aura should be applied to a target and not self
-
+            targetedAura : "target",// "self", "target", "all" // Aura - if the aura should be applied to a target and not self
```

Comment:
I don't really have a mind to go through the effects again and select the correct value here. This should work similarly to normal targeted effects:

`targetedAura` is a boolean, if true, and the spell/miracle designates self/you as the target, it applies to yourself automatically. Such as, say, Aetheric Arms/Armour.

`targetedAura` might be worth renaming as well, maybe `appliedAura`.

Reply:
there is a big chance, that i dont understand something related to auras. I believe there are few kinds of auras. In terms of who should be affected:
- everyone in range except token "owning" aura, 
- everyone in range including token "owning" aura - and this is the case, that i'm not sure who to achieve.

and in terms of how to apply aura: 
- whenever an item is 'invoked' - spells/miracles, maybe some other manual effects (didnt find a usecase yet (potion?))
- whenever an item is "owned" - traits, equipped weapones. 

for first case (spells), i assume that ```applicationData.documentType``` should be ```Item``` and for second (traits) it should be ```Actor```?